cmake-modules/global_compile_options.cmake

```diff
@@ -4,7 +4,7 @@ if(MSVC)
   endif()
 
   #https://stackoverflow.com/questions/37527946/warning-unreferenced-inline-function-has-been-removed
-  add_compile_options(/W4 ${WARNINGS_AS_ERRORS_FLAG} /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 /analyze)
```

Comment:
I don't recommend this as the warnings at /W4 catch lots of real bugs. Note that /W4 is required by SDL policy.